tests/appsec/iast_packages/test_packages.py

```diff
@@ -943,7 +943,7 @@ def test_flask_packages_patched(package, venv):
     [
         package
         for package in PACKAGES
-        if package.test_propagation and package.name != "rsa"  # the "rsa" mode fails reliably
+        if package.test_propagation and package.name not in ("rsa", "cryptography")  # reliably fails on main
```

Comment:
These tests have some custom logic to fail or xfail that allows us to see when we've reached enough propagation coverage so the modules start to behave as expected.

Seeing this is not some binary thing, we prefer changing (relaxing) the logic allowing for the test to run but not fail the CI.

Does that sound good?
Working on it here: https://github.com/DataDog/dd-trace-py/pull/9838